frontend/packages/integration-tests-cypress/tests/events/events.cy.ts

```diff
@@ -1,5 +1,11 @@
 import { testName, checkErrors } from '../../support';
 
+const events = {
+  event: '.co-sysevent',
+  total: '.co-sysevent-stream__totals',
+  warning: '.co-sysevent--warning',
```

Comment:
That's a good point.  I opted not to in this case since this is a custom (non-PatternFly component) that relies on these existing classnames for styling.  Happy to change it if you feel I should.

Reply:
I would second that we use a data attribute rather than class names since it's recommended best practice per [Cypress docs](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements).

Reply:
Updated.